File: app/src/main/java/com/delivery/deliveryapp/models/Restaurant.java
package com.delivery.deliveryapp.models;

import java.io.Serializable;
import java.util.ArrayList;

public class Restaurant implements Serializable{

    private String name;
    private ArrayList<Menu> menus;

    public Restaurant(String name)
    {
        this.name = name;
        this.menus = new ArrayList<Menu>();
    }

    public Restaurant(String name, ArrayList<Menu> menus)
    {
        super();
        this.name = name;
        this.menus = menus;
    }

    public String getName() {return this.name;}
    public ArrayList<Menu> getMenus() {return this.menus;}

    public void addMenu(Menu menu) {this.menus.add(menu);}

}

File: app/src/main/java/com/delivery/deliveryapp/models/Restaurant.kt
package com.delivery.deliveryapp.models

import java.io.Serializable
import java.util.ArrayList

class Restaurant(val name: String) : Serializable {

    var menus: ArrayList<Menu>

    init{
        this.menus = ArrayList()
    }

    constructor(name: String, menus: ArrayList<Menu>) : this(name) {
        this.menus = menus
    }

    fun addMenu(menu: Menu) {
        this.menus.add(menu)
    }

}
